account_payment_order_notification/data/mail_template_data.xml

```diff
@@ -30,14 +30,18 @@
                 <td>Amount</td>
             </tr>
         </thead>
-        % for payment_line in object.payment_line_ids:
-            <tr>
-                <td>${payment_line.communication}</td>
-                <td>${payment_line.date}</td>
-                <td>${format_amount(payment_line.amount_currency, payment_line.currency_id)}</td>
-            </tr>
-        % endfor
     </table>
+    % for payment_line in object.payment_line_ids:
+        <table>
```

Comment:
Why a new table? The previous problem seems to be the absence of the `<tbody>` element?

Reply:
I would have to check specifically why (html formatting "breaks" it).

I have found a "quick" example of odoo core in which there is also a similar error: https://github.com/odoo/odoo/blob/14.0/addons/gamification/data/goal_base.xml#L26-L226, however, in `sale` there is this example https://github.com/odoo/odoo/blob/14.0/addons/sale/data/mail_data.xml#L119-L294 in which it uses several tables.

Reply:
But putting `tbody` should solve it.

Reply:
The second one is not OK, as the width of the header is not going to be adapted to the content of the rest. Anyway, ignore the editor. The important thing is that the content gets to the mail correctly.